client/apps/game/src/ui/layouts/world.tsx

```diff
@@ -259,38 +259,6 @@ export const World = ({ backgroundImage }: { backgroundImage: string }) => {
     syncUnsyncedStructures();
   }, [account.account, playerStructures, fetchedStructures, syncStructures, setSyncedStructures]);
 
-  const syncQuestTiles = useCallback(
```

Comment:
can you make sure to remove the quests = useQuests as it' snot used and would provoke rerender at every update